$OpenBSD: patch-build_xml,v 1.6 2013/03/19 23:43:06 jeremy Exp $

Don't install the jruby launcher when running the regress specs,
as it fails due to a make error and it isn't necessary for the
regress tests.

--- build.xml.orig	Tue Aug  7 07:49:40 2012
+++ build.xml	Tue Aug  7 12:37:33 2012
@@ -702,7 +702,7 @@ other than ASM, which is rewritten to avoid conflicts.
         </emma>
     </target>
 
-    <target name="install-gems" depends="install-dev-gems,install-jruby-launcher-gem"/>
+    <target name="install-gems" depends="install-dev-gems"/>
 
     <condition property="dev.gems.installed">
         <uptodate>
